Loading cache...done.
Loaded 1825 entries from dependency cache.
Parsing recipes...done.
Parsing of 1043 .bb files complete (1041 cached, 2 parsed). 1828 targets, 209 skipped, 0 masked, 0 errors.
NOTE: Resolving any missing task queue dependencies

Build Configuration:
BB_VERSION           = "1.42.0"
BUILD_SYS            = "x86_64-linux"
NATIVELSBSTRING      = "universal"
TARGET_SYS           = "i586-deby-linux"
MACHINE              = "qemux86"
DISTRO               = "deby-tiny"
DISTRO_VERSION       = "10.0"
TUNE_FEATURES        = "m32 i586"
TARGET_FPU           = ""
meta                 
meta-poky            = "warrior:79a850a10a4b88a6d20d607b322542f947874323"
meta-debian          = "warrior:2fa6d8525f0464064f8764856015185a0c4e5411"

Initialising tasks...done.
Sstate summary: Wanted 5 Found 0 Missed 5 Current 126 (0% match, 96% complete)
NOTE: Executing SetScene Tasks
NOTE: Executing RunQueue Tasks
NOTE: Running task 740 of 767 (/home/deby/poky/meta-debian/recipes-debian/python/python-setuptools_debian.bb:do_prepare_recipe_sysroot)
NOTE: recipe python-setuptools-40.8.0-r0: task do_prepare_recipe_sysroot: Started
NOTE: recipe python-setuptools-40.8.0-r0: task do_prepare_recipe_sysroot: Succeeded
NOTE: Running task 759 of 767 (/home/deby/poky/meta-debian/recipes-debian/python/python-setuptools_debian.bb:do_configure)
NOTE: recipe python-setuptools-40.8.0-r0: task do_configure: Started
NOTE: recipe python-setuptools-40.8.0-r0: task do_configure: Succeeded
NOTE: Running task 760 of 767 (/home/deby/poky/meta-debian/recipes-debian/python/python-setuptools_debian.bb:do_compile)
NOTE: recipe python-setuptools-40.8.0-r0: task do_compile: Started
NOTE: recipe python-setuptools-40.8.0-r0: task do_compile: Succeeded
NOTE: Running task 761 of 767 (/home/deby/poky/meta-debian/recipes-debian/python/python-setuptools_debian.bb:do_install)
NOTE: recipe python-setuptools-40.8.0-r0: task do_install: Started
NOTE: recipe python-setuptools-40.8.0-r0: task do_install: Succeeded
NOTE: Running task 762 of 767 (/home/deby/poky/meta-debian/recipes-debian/python/python-setuptools_debian.bb:do_package)
NOTE: Running task 763 of 767 (/home/deby/poky/meta-debian/recipes-debian/python/python-setuptools_debian.bb:do_populate_sysroot)
NOTE: recipe python-setuptools-40.8.0-r0: task do_package: Started
NOTE: recipe python-setuptools-40.8.0-r0: task do_populate_sysroot: Started
NOTE: recipe python-setuptools-40.8.0-r0: task do_populate_sysroot: Succeeded
NOTE: recipe python-setuptools-40.8.0-r0: task do_package: Succeeded
NOTE: Running task 764 of 767 (/home/deby/poky/meta-debian/recipes-debian/python/python-setuptools_debian.bb:do_packagedata)
NOTE: recipe python-setuptools-40.8.0-r0: task do_packagedata: Started
NOTE: recipe python-setuptools-40.8.0-r0: task do_packagedata: Succeeded
NOTE: Running task 765 of 767 (/home/deby/poky/meta-debian/recipes-debian/python/python-setuptools_debian.bb:do_package_write_deb)
NOTE: Running task 766 of 767 (/home/deby/poky/meta-debian/recipes-debian/python/python-setuptools_debian.bb:do_package_qa)
NOTE: recipe python-setuptools-40.8.0-r0: task do_package_write_deb: Started
NOTE: recipe python-setuptools-40.8.0-r0: task do_package_qa: Started
NOTE: recipe python-setuptools-40.8.0-r0: task do_package_qa: Succeeded
NOTE: recipe python-setuptools-40.8.0-r0: task do_package_write_deb: Succeeded
NOTE: Running noexec task 767 of 767 (/home/deby/poky/meta-debian/recipes-debian/python/python-setuptools_debian.bb:do_build)
NOTE: Tasks Summary: Attempted 767 tasks of which 757 didn't need to be rerun and all succeeded.
